test/trans.js

```diff
@@ -0,0 +1,129 @@
+var fs = require("fs");
+var assert = require("assert");
+var {Context} = require('../lib/context');
+var {Environment} = require('../lib/environment');
+var {Template} = require("../lib/template");
+var {TemplateSyntaxError} = require('../lib/errors');
+var ilib = require("../../ilib");
```

Comment:
> Is there some way to add to the path where the test framework looks for packages?

yeah, there's a command line arg:

    ringo --modules /path/to/packages/

or you can install the package globally with `rp install --global`